tez-tests/src/test/java/org/apache/tez/test/TestTezJobs.java

```diff
@@ -164,10 +164,35 @@ public static void tearDown() {
   public void testHashJoinExample() throws Exception {
     HashJoinExample hashJoinExample = new HashJoinExample();
     hashJoinExample.setConf(new Configuration(mrrTezCluster.getConfig()));
-    Path stagingDirPath = new Path("/tmp/tez-staging-dir");
-    Path inPath1 = new Path("/tmp/hashJoin/inPath1");
-    Path inPath2 = new Path("/tmp/hashJoin/inPath2");
-    Path outPath = new Path("/tmp/hashJoin/outPath");
+    runHashJoinExample(hashJoinExample);
+  }
+
+  @Test(timeout = 60000)
+  public void testHashJoinExampleWithLogPattern() throws Exception {
+    HashJoinExample hashJoinExample = new HashJoinExample();
+
+    Configuration patternConfig = new Configuration(mrrTezCluster.getConfig());
+
+    patternConfig.set(TezConfiguration.TEZ_AM_LOG_LEVEL, "debug");
+    patternConfig.set(TezConfiguration.TEZ_TASK_LOG_LEVEL, "debug");
+    patternConfig.set(TezConfiguration.TEZ_LOG_PATTERN_LAYOUT_AM,
+        "%d{ISO8601} [%p] [%t (queryId=%X{queryId} dag=%X{dagId})] |%c{2}|: %m%n");
+    patternConfig.set(TezConfiguration.TEZ_LOG_PATTERN_LAYOUT_TASK,
```

Comment:
Can you check for the case when the value can be reset back to original value? Plz refer to my comment earlier in TezConfiguration.

Reply:
yeah, added testcase